CMakeLists.txt

```diff
@@ -112,7 +114,7 @@ if(WIN32)
   if(MSVC60)
   set(WINVER 0x0400)
   else()
-  set(WINVER 0x0500)
+  set(WINVER 0x0600)
```

Comment:
I'm not familiar with Windows' versioning. Can you explain this change?

Reply:
The [One-Time Initialization](https://docs.microsoft.com/en-us/windows/win32/sync/one-time-initialization) functions were introduced in Windows Vista. The `WINVER` value for Windows Vista is [0x0600](https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=vs-2019).